backend/requirements.txt

```diff
@@ -38,9 +38,9 @@ jsonpatch==1.33
 jsonpointer==3.0.0
```

Comment:
Potential issue: langchain-core was updated from 0.3.6 to 0.3.25, which may cause compatibility issues.